src/middleware/jwtValidator.js

```diff
@@ -12,17 +10,16 @@ const checkToken = (req, res, next) => {
 
   if (token) {
     const { JWT_KEY } = process.env;
-    jwt.verify(token, JWT_KEY, (err, decoded) => {
+    jwt.verify(token, JWT_KEY, (err, user) => {
       if (err) {
-        setError(401, 'Token is not valid');
-      } else {
-        req.decoded = decoded;
+        return errorResponse(res, 401, 'Token is not valid');
```

Comment:
We could change the message to 'Invalid Token'
return errorResponse(res, 401, 'Invalid Token');